backend/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.5.6</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.datmt</groupId>
+    <artifactId>keycloak-springboot-microservice</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>keycloak-springboot-authentication</name>
+    <description>Spring boot keycloak authen</description>
+
+    <properties>
+        <java.version>11</java.version>
+        <keycloak.version>15.0.2</keycloak.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
For each dependency, please add a small documentation describing why we need it. Further, remove the redundant ones if any.